src/utilities/core-utils.test.ts

```diff
@@ -91,6 +91,127 @@ describe("CoreUtils", () => {
 
     // #endregion getRandomEnum
 
+    // -----------------------------------------------------------------------------------------
+    // #region numericEnumToPojo
+    // -----------------------------------------------------------------------------------------
+
+    describe("numericEnumToPojo ", () => {
+        test("when given numeric enum, returns an object of values", () => {
+            // Arrange
+            enum testEnum {
+                first,
```

Comment:
While the implicit behavior of an enum is to be numeric, would you update this to have explicit numeric values so it's clearer to a reader?